Makefile

```diff
@@ -185,19 +185,21 @@ endif
 $(PREP_LINK_SCRIPT) : $(LINK_SCRIPT)
 	$(VERBOSE) $(CC) $(AFLAGS) -E -P -C -x c $< -o $@
 
-$(TARGET): $(OBJS) $(PREP_LINK_SCRIPT)
+$(TARGET_DEBUG): $(OBJS) $(PREP_LINK_SCRIPT)
 	@echo "LD " $@
 	$(VERBOSE) $(LD) -T $(PREP_LINK_SCRIPT) -o $@ $(OBJS) $(PFMLIB_LINKER_FLAGS)
 	@echo "GEN " $(SYMBOLS_NAME).S
-	$(VERBOSE) $(NM) -p --format=posix $(TARGET) | $(PYTHON) $(SYMBOLS_DIR)/$(SYMBOLS_TOOL)
+	$(VERBOSE) $(NM) -p --format=posix $@ | $(PYTHON) $(SYMBOLS_DIR)/$(SYMBOLS_TOOL)
 	@echo "CC " $(SYMBOLS_NAME).S
 	$(VERBOSE) $(CC) -c -o $(SYMBOLS_NAME).o $(AFLAGS) $(SYMBOLS_NAME).S
 	$(VERBOSE) rm -rf $(SYMBOLS_NAME).S
-	@echo "LD " $(TARGET) $(SYMBOLS_NAME).o
+	@echo "LD " $@ $(SYMBOLS_NAME).o
 	$(VERBOSE) $(LD) -T $(PREP_LINK_SCRIPT) -o $@ $(OBJS) $(PFMLIB_LINKER_FLAGS) $(SYMBOLS_NAME).o
+
+$(TARGET): $(TARGET_DEBUG)
 	@echo "STRIP"
-	$(VERBOSE) $(OBJCOPY) --only-keep-debug $(TARGET) $(TARGET_DEBUG)
-	$(VERBOSE) $(STRIP) $(STRIP_OPTS) $(TARGET)
+	$(VERBOSE) cp $< $@
```

Comment:
Instead of making a copy beforehand, simply make use of the `-o ...` option of `strip`:
```make
$(VERBOSE) $(STRIP) $(STRIP_OPTS) -o $@ $<
```

Reply:
Fixed. Thanks.